README.md

```diff
@@ -53,244 +65,68 @@ coastal_nomangrove_minus_mangrove__rp_100__rcp_baseline__epoch_2010__conf_None.t
-```
-
-or using Docker Compose `snakemake` service, first edit the `snakemake` command to point
-to the relevant `Snakefile` for a pipeline:
-
-```
-snakemake:
-    image: ghcr.io/nismod/gri-snakemake:0.1
-    ...
-    command:
-      [
-        ...
-        "-s",
-        "/opt/etl/pipelines/iris/Snakefile"
-      ]
-```
-
-then run:
-
-```bash
-docker compose -f docker-compose-dev.yaml run snakemake
-```
-
-#### Raster Tileserver Ingester
-
-Update `docker-compose-dev.yaml` `raster-tile-ingester` block as required for a
-dataset (after running its pre-processing and Snakemake pipeline)
-
-For more information see [raster-tile-ingester](containers/raster-tile-ingester/README.md)
-
-```bash
-docker-compose -f docker-compose-dev.yaml run raster-tile-ingester
-```
-
-## Build - Docker
-
-Repository: `ghcr.io/nismod`
-
-The application is built with several 'services', each facilitated by a running
-docker container.
-
-Services:
-
-- Web server (nginx) `ghcr.io/nismod/gri-web-server`
-- Vector tileserver (tileserver-gl) `ghcr.io/nismod/gri-vector-tileserver`
-- Backend / API (bespoke Python app for vector data and raster tiles (+meta)) `ghcr.io/nismod/gri-backend`
-- API Database (PostgreSQL with PostGIS serves backend) (Dev only)
-- Tiles Database (MySQL serves tile ingester and backend /tiles endpoints) (Dev only)
+## Environment
 
-The services are orchestrated using `docker compose`. N.B. The app was built
-with docker engine version 20.10.16 and compose version 2.5.0. It may not work
-with other versions.
+Environment variables for the various services (and the ETL workflow) are
+stored in env files. Example files are given in `envs/dev-example`. These can
+be placed in `envs/dev` to get started.
 
-The `compose.yml` files contain service names and definitions, and paths to
-build contexts (all located within `containers/`).
+Production env files should be placed in `envs/prod`.
 
-### docker-compose-dev.yaml
+## Deploy
 
-Containers used for local development of the entire stack, including ETL.
+To deploy the stack we use the `docker compose` tool.
 
-### docker-compose-prod.yaml
+### Development
 
-Used to run local builds of Production containers.
+To deploy locally, `docker compose -f docker-compose-dev.yaml up <desired services>`.
+The set of services can include:
+- tiles-db
+- db
+- backend
+- web-server
+- vector-tileserver
+- redis
+- irv-autopkg-work
+- irv-autopkg-api
+- traefik
 
-### docker-compose-deploy.prod
+If you're running your own FE development server, or connecting to a remotely
+hosted database, or not using the autopackage API, you may not need all these
+services.
 
-Used for deployment of containers into a production environment.
+For example, when running a FE development server to add a new raster layer the
+following should suffice:
+`docker compose -f docker-compose-dev.yaml up tiles-db db backend`
 
-## Deploy
+N.B. If you find that the `backend` service is complaining that the
```

Comment:
Rather than this being an aside note, I'd state more explicitly that the database needs to be initialised the first time the `db` service is ran - since we know that otherwise, by definition, the `backend` service will complain.

Reply:
We just need to be clear that this clears the database so that someone doesn't lose data. Also wondering what the risk is of people accidentally running all services in the compose file on an already initialised project. When running `docker compose -f FILE up`, does it spin up all services indiscriminately? Seems like an easy way to lose data!